bin/config.js

```diff
@@ -1,19 +1,11 @@
-const rc = require('rc')
+const fsproxy = require('fs-proxy')
 
-module.exports = rc('cordlr', {
```

Comment:
Hmm, one thing I have noticed is that `rc` allowed you to use command line options:

```js
cordlr --token='...' --prefix='...'
```

Which I use heavily in development, and for just fast configuring (when you are lazy).